docs/SCA_temperature_voltage_unit_conversion.md

```diff
@@ -0,0 +1,49 @@
+Table of Contents
+=================
+
+   * [ADC counts to voltage conversion](#adc-counts-to-voltage-conversion)
+      * [PT100 Sensors](#pt100-sensors)
+      * [SCA Internal temperature](#sca-internal-temperature)
+      * [Conversion of ADC for Voltage sensors](#conversion-of-adc-for-voltage-sensors)
+      * [Reference](#reference)
+
+Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
+
+# ADC counts to voltage conversion
+
+The output values are 12 bit ADC and the voltage value varies from 0 to 1 V. Thus, each bit corresponds to 1/2^{12} =~ 0.244 mV.
+
+## PT100 Sensors 
+
+- ADC count = 44 (say)
+- Corresponding voltage = 44 x 0.244 mV = 10.736 mV
+- We are measuring temperature using the platenium resistor from which 100 $\mu$A current is passing.
+- Resistance = V/I = 10.736mV / 100 $\mu$ A = 107.36  $\Omega$
+- Check the temperature corresponding to the 107.36 $\Omega$ resistance in the R - T table [1]. This corresponds to ~19 $^0 C$.
```

Comment:
Markdown format should support `LaTeX`:

https://docs.mattermost.com/help/messaging/formatting-text.html#math-formulas

Might be clearly instead of doing this style of example to instead put in the mathematical formulas.  Then any developer has an easy quick reference to use instead of having to walk through the example.

Reply:
Of course 

```latex
X_k = \sum_{n=0}^{2N-1} x_n \cos \left[\frac{\pi}{N} \left(n+\frac{1}{2}+\frac{N}{2}\right) \left(k+\frac{1}{2}\right) \right]
```